src/main/java/uk/gov/hmcts/cmc/claimstore/documents/content/ClaimIssueReceiptContentProvider.java

```diff
@@ -0,0 +1,59 @@
+package uk.gov.hmcts.cmc.claimstore.documents.content;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+import uk.gov.hmcts.cmc.claimstore.services.staff.content.ClaimContentProvider;
+import uk.gov.hmcts.cmc.claimstore.services.staff.content.ClaimantContentProvider;
+import uk.gov.hmcts.cmc.claimstore.services.staff.content.PersonContentProvider;
+import uk.gov.hmcts.cmc.domain.models.Claim;
+import uk.gov.hmcts.cmc.domain.models.otherparty.TheirDetails;
+import uk.gov.hmcts.cmc.domain.utils.PartyUtils;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import static java.util.Objects.requireNonNull;
+
+@Component
+public class ClaimIssueReceiptContentProvider {
+
+    private final ClaimantContentProvider claimantContentProvider;
+    private final PersonContentProvider personContentProvider;
+    private final ClaimContentProvider claimContentProvider;
+
+    @Autowired
+    public ClaimIssueReceiptContentProvider(
+        final ClaimantContentProvider claimantContentProvider,
+        final PersonContentProvider personContentProvider,
+        final ClaimContentProvider claimContentProvider)
+    {
+        this.claimantContentProvider = claimantContentProvider;
+        this.personContentProvider = personContentProvider;
+        this.claimContentProvider = claimContentProvider;
+    }
+    public Map<String, Object> createContent(final Claim claim) {
+        requireNonNull(claim);
+
+        Map<String, Object> map = new HashMap<>();
+
+        map.put("claimant", claimantContentProvider.createContent(
+            claim.getClaimData().getClaimant(), claim.getSubmitterEmail())
+        );
+
+        TheirDetails defendant = claim.getClaimData().getDefendant();
+
+        map.put("defendant", personContentProvider.createContent(
+            PartyUtils.getType(defendant),
+            defendant.getName(),
+            defendant.getAddress(),
+            null,
```

Comment:
Why this is a null? Don't we display correspondence address?

Reply:
Explained in in person "TheirDetails" doesn't not contain a correspondence address field